include/library/spdm_return_status.h

```diff
@@ -42,7 +42,7 @@ typedef uint32_t libspdm_return_t;
 #define LIBSPDM_SOURCE_CORE 0x01
 #define LIBSPDM_SOURCE_CRYPTO 0x02
 #define LIBSPDM_SOURCE_CERT_PARSE 0x03
-#define LIBSPSM_SOURCE_TRANSPORT 0x04
+#define LIBSPDM_SOURCE_TRANSPORT 0x04
```

Comment:
You also need to keep the `#define` for the misspelled macro. We can get rid of it in a future release.

Reply:
Thanks. I have made it back and added comment.